utils/tokens.tsx

```diff
@@ -439,27 +444,51 @@ export const getNfts = async (
   return await getNftsFromHolaplex(ownerPk, connection)
 }
 
+// TODO: Remove when upgrade spl-token
```

Comment:
What's the upgrade referred to here? Sorry if I'm out of the loop

Reply:
Ah this is left over, I was going to upgrade spl-token but ended up not having to.